src/compile/selection/point.ts

```diff
@@ -1,11 +1,49 @@
-import {Stream} from 'vega';
+import {Signal, Stream} from 'vega';
 import {stringValue} from 'vega-util';
-import {SelectionCompiler, TUPLE, unitName} from '.';
+import {SelectionCompiler, TUPLE, isTimerSelection, unitName} from '.';
 import {SELECTION_ID} from '../../selection';
 import {vals} from '../../util';
 import {BRUSH} from './interval';
 import {TUPLE_FIELDS} from './project';
 
+export const CURR = '_curr';
+
+const animationSignals = (selectionName: string, scaleName: string): Signal[] => {
+  return [
+    // timer signals
+    {
+      name: 'anim_clock',
```

Comment:
Can we avoid hardcoding signal names and instead make use of exported CONSTs? That'll make any future refactoring just a little bit easier.

Reply:
addressed by josh in https://github.com/vega/vega-lite/pull/8921/commits/2d55ccf7c77f398bdf51d8c8b2e9c31d40bfef66